src/client/eppo-precomputed-client.ts

```diff
@@ -324,39 +324,38 @@ export default class EppoPrecomputedClient {
     flagKey: string,
     defaultValue: string,
   ): Omit<IAssignmentDetails<string>, 'evaluationDetails'> {
-    const banditEvaluation = this.getPrecomputedBandit(flagKey);
-
-    if (banditEvaluation == null) {
-      logger.warn(`${loggerPrefix} No assigned variation. Bandit not found: ${flagKey}`);
+    const precomputedFlag = this.getPrecomputedFlag(flagKey);
+    if (!precomputedFlag) {
+      logger.warn(`${loggerPrefix} No assigned variation. Flag not found: ${flagKey}`);
       return { variation: defaultValue, action: null };
```

Comment:
Returning this if the precomputed flag is not found, otherwise returning the precomputed flag assignment if the bandit could not be evaluated. Makes sense